src/ui/public/styles/control_group.less

```diff
@@ -24,7 +24,7 @@
   }
 
   button {
-    padding: @padding-base-vertical @padding-base-horizontal;
+    padding: 4px 10px;
```

Comment:
These numbers should probably be variables, almost certainly existing variables.